### PR TITLE
Replaced MessageManager deprecated method - addnotice

### PR DESCRIPTION
**Pull request by rengaraj-anandan**
### Description

- Replaced the deprecated method addNotice to addNoticeMessage in Magento modules.
- Fixed appropriate PHPUnit tests

### Manual testing scenarios

- Only replaced the deprecated methods.

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/ImportExport/Controller/Adminhtml/Import/Index.php [hunk 1]
```diff
@@ -5,11 +5,15 @@
...
+use Magento\Framework\App\Action\HttpGetActionInterface;
+use Magento\Framework\App\Action\HttpPostActionInterface;
 use Magento\Framework\Controller\ResultFactory;
+use Magento\ImportExport\Controller\Adminhtml\Import as ImportController;
 
-class Index extends ImportController implements HttpGetActionInterface
+/**
+ * Import Index Controller
+ */
+class Index extends ImportController implements HttpGetActionInterface, HttpPostActionInterface
```
Comment (ihor-sviziev) [inline app/code/Magento/ImportExport/Controller/Adminhtml/Import/Index.php]: Why should this action handle post requests?

File: app/code/Magento/Reports/Controller/Adminhtml/Report/AbstractReport.php [hunk 2]
```diff
@@ -132,16 +133,16 @@ public function _initReportAction($blocks)
...
+     * @param  string $refreshCode
      * @return $this
      */
     protected function _showLastExecutionTime($flagCode, $refreshCode)
     {
         $flag = $this->_objectManager->create(\Magento\Reports\Model\Flag::class)
             ->setReportFlagCode($flagCode)
             ->loadSelf();
-        $updatedAt = __('Never');
+        $updatedAt = 'Never';
```
Comment (ihor-sviziev) [inline app/code/Magento/Reports/Controller/Adminhtml/Report/AbstractReport.php]: Why did you removed translation here?

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/ImportExport/Controller/Adminhtml/Import/Index.php (hunk 1 above)
Comment (rengaraj-anandan) [reply app/code/Magento/ImportExport/Controller/Adminhtml/Import/Index.php]: Thanks for review @ihor-sviziev 

Actually I'm still progressing on the pull request facing few more issues regards the replace addnotice deprecated methods. I will fix that too and then let you know.

Comment (ihor-sviziev) [reply app/code/Magento/ImportExport/Controller/Adminhtml/Import/Index.php]: Ok, let me know once you’ll finish

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE, Integration Tests

**Review by ihor-sviziev — DISMISSED** [review]
Hi @rengaraj-anandan,
According to failing functional & integration tests - looks like your changed did some regression in functionality.

Could you update your PR?

**Comment by ihor-sviziev** [comment]
@rengaraj-anandan I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Pull request closed without merging**